cdk/package.json

```diff
@@ -9,7 +9,7 @@
     "generate": "cdk synth --path-metadata false --version-reporting false"
   },
   "devDependencies": {
-    "@guardian/cdk": "41.0.1",
+    "@guardian/cdk": "41.1.0-beta.1",
```

Comment:
We can use a standard version of the library once https://github.com/guardian/cdk/pull/1211 has been merged/released.

Reply:
https://github.com/guardian/prism/pull/319